includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -904,9 +904,9 @@ private function get_stylesheet_priority( DOMNode $node ) {
 			$element_id      = (string) $node->getAttribute( 'id' );
 			$schemeless_href = $remove_url_scheme( $node->getAttribute( 'href' ) );
 			$is_plugin_asset = (
-				0 === strpos( $schemeless_href, $remove_url_scheme( trailingslashit( plugins_url( WP_PLUGIN_DIR ) ) ) )
```

Comment:
The plugin prioritization did never work correctly, I think. Not only does the non-MU version use the `_DIR` instead of the `_URL`, they are also wrapped in `plugins_url()`.

`plugins_url()` assumes its argument is a relative path to be appended to the plugin folder URL. So the result was something like `//example.org/wp-content/plugins/Users/alain/wp-content/plugins/`

Reply:
Good catch.